test/python/conftest.py

```diff
@@ -86,19 +93,19 @@ def light_project(original_light_project, deprecated_light_project, request):
 #########################
 # Projects with cartridge
 #########################
-@pytest.fixture(scope="module")
```

Comment:
Are you ready for it? It could significantly slow down testing process

Reply:
Yes, I sure that `project` fixtures should have `function` scope to prevent accidental changing project files.

This fixture had `module` scope because it was used for `docker_image` fixture with `module` scope (it was used for testing image contents and e2e).

Now, I use light project to test image/archive contents and use `project_with_cartridge` fixture only for e2e tests (one archive per test).

Reply:
agree